static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain-sections.js

```diff
@@ -0,0 +1,10 @@
+define(function () {
+    var sections = ['politics', 'world', 'business', 'commentisfree'];
+
+    function getSection(section) {
+        section = section.toLowerCase();
+        return /news/.test(section) || sections.indexOf(section) !== -1 ? 1 : 2;
```

Comment:
yes, I really struggled with this. I used the same vocabulary than Outbrain, but yours is clearer.